src/batched/KokkosBatched_Util.hpp

```diff
@@ -3,7 +3,7 @@
 
 /// \author Kyungjoo Kim (kyukim@sandia.gov)
 
-#define __KOKKOSBATCHED_PROMOTION__ 1
+//#define __KOKKOSBATCHED_PROMOTION__ 1
```

Comment:
These changes are already in master, right ? How will this merge work ? I am just asking as I don't know.

Reply:
I think that some commits were put in my branch by mistake when I rebased my code to a more recent version of develop.
I will investigate how to clean this.

I also have a problem with running the unit-test with Cuda and Serial instantiated at the same time. The Cuda unit-tests run fine on there own and OpenMP+Serial runs fine too but combinations of Cuda+OpenMP or Cuda+Serial are crashing...

Reply:
The crash do not happen in the perf-test though for some reason?